docs/spec/channel.md

```diff
@@ -15,15 +15,14 @@ A typical channel consists of a _Controller_ and a _Dispatcher_ pod.
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
-interpreted as described in [RFC2119][rfc2119].
+interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 ## Channel Spec Parts
 
 ### Control Plane
 
 Each Channel implementation is backed by its own CRD (e.g. `InMemoryChannel` or
-`KafkaChannel`). Unlike in earlier versions there is no concept for a _default
-channel_, using the different CRDs. Below is an example for a `KafkaChannel`
+`KafkaChannel`). Below is an example for a `KafkaChannel`
 object:
 
```

Comment:
Is the structure of .yaml files MUST requirement? How are we going to test it? By parsing CRD yaml files ? Any naming convention expected like xyzchannels.messaging.knative.dev?

Reply:
The actual YAML structure can be different, as long as at least those permissions are granted. However, for anything checked into knative itself, we will likely want the style to match too.

Added something for the naming convention in #1673.